android/build.gradle

```
@@ -28,6 +28,10 @@ android {
 buildscript {
     ext {
         kotlinVersion = '1.4.31'
```

I think the default version should be updated to a version that works with latest React Native versions
```suggestion
        kotlinVersion = '1.6.21'
```
The other changes look good to me.